### PR TITLE
Make pg-related edits to get views to identically render

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

This PR aims to get the views a) all loading and b) in agreement with a fresh seed. (spoiler: it does)

After this, we'll be doing archived patient and tests, and then hopefully we're in the clear.

Note that generally we are relying on system tests to confirm that behavior is generally the same, which aren't ported yet, so I think you can generally take these code changes at more or less face value; we'll do a more involved click-around review when the feature branch is really ready to go I think.

This pull request makes the following changes:
* mostly small place changes to get stuff to properly render
* a slightly more involved change to changelog to accommodate timestamps coming out of the database differently

It relates to the following issue #s: 
* Bumps #2072 

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

**Comment by colinxfleming** [comment]
Views _mostly_ identically render now, with three exceptions (one of which doesn't really count yet):
* changelog - mongoid and created_at are showing up
* changelog - fund_pledged_at is showing up with full 
* export - errors, but I think it's because archived patient isn't ported, so not dealing with that yet

I'll fix the first two and then punt this up.

**Comment by colinxfleming** [comment]
(There are a couple minor differences like ordering differences but I think that's more because of migrate_to_pg than it is actual logic changes.)

File: app/controllers/accountants_controller.rb [hunk 1]
```diff
@@ -37,8 +37,8 @@ def paginate_results(results)
 
   def pledged_patients
     Patient.where(pledge_sent: true,
-                  :initial_call_date.gte => 6.months.ago)
```
Comment (lomky) [inline app/controllers/accountants_controller.rb:40]: I will never miss trying to wrap my head around gte/lte date math 🎉

**Review by lomky — APPROVED** [review]
image

**Pull request merged**